Creating blogpost instruction update (#3491)
sourcegraph/handbook: content/departments/marketing/content/creating_blog_posts.md

The data between the `---` is called front matter and is used to provide post metadata. Important to note about this metadata, is that:

> - The `description` field is optional and only needed if you want to craft the description for your post on the blog the index page.

Reviewer: I think this should be required since it's used as an excerpt for blog posts on the index.

```suggestion
The `description` field is used as an excerpt for your post on the blog the index page.
```

> - The `description` field is optional and only needed if you want to craft the description for your post on the blog the index page.

Reviewer: I think this should be required since it's used as an excerpt for blog posts on the index.

```suggestion
The `description` field is used as an excerpt for your post on the blog the index page.
```

Uses Bootstrap for responsive sizing and adequate whitespace between adjacent elements, and that only Sourcegraph videos are shown on the end screen.

```html
<div className="container my-4 video-embed embed-responsive embed-responsive-16by9">
  <iframe
    className="embed-responsive-item"
    src="https://www.youtube-nocookie.com/embed/${YOUTUBE_ID}?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0"

> allowFullScreen=""

Reviewer: ```suggestion
    allowFullScreen=""
```

Uses Bootstrap for responsive sizing and adequate whitespace between adjacent elements, and that only Sourcegraph videos are shown on the end screen.

```html
<div className="container my-4 video-embed embed-responsive embed-responsive-16by9">
  <iframe
    className="embed-responsive-item"
    src="https://www.youtube-nocookie.com/embed/${YOUTUBE_ID}?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0"

>     allowfullscreen=""

Reviewer: ```suggestion
allowFullScreen=""
```
